content/sensu-go/5.14/getting-started/learn-sensu.md

```diff
@@ -308,20 +301,19 @@ Open the `influx-handler.json` handler definition provided with the sandbox, and
 "runtime_assets": ["sensu-influxdb-handler"]
 {{< /highlight >}}
 
-Now you can use sensuctl to create the `influx-db` handler.
+Now you can use sensuctl to create the `influx-db` handler:
 
 {{< highlight shell >}}
 sensuctl create --file influx-handler.json
 {{< /highlight >}}
 
-We can use sensuctl to confirm that the handler has been created successfully.
+Use sensuctl to confirm that the handler was created successfully.
 
 {{< highlight shell >}}
 sensuctl handler list
 {{< /highlight >}}
 
-You should see the `influx-db` handler.
-(If you've completed [lesson \#2](#lesson-2-pipe-keepalive-events-into-slack), you'll also see the `keepalive` handler.)
+The `influx-db` handler should be listed. If you completed [lesson \#2](#lesson-2-pipe-keepalive-events-into-slack), you'll also see the `keepalive` handler.
 
 **4. Create a check to monitor Nginx**
 
```

Comment:
The instructions here do not tell you where to type `metrics-curl.rb` (I guessed that it's a handler, and my response for `sensuctl create --file curl_timings-check.json` matches the one in the lesson, but I'm still not sure I did this step correctly).

Reply:
`metrics-curl.rb` is a check plugin which is referenced in the value of the `command` field in `curl_timings-check.json`. See https://github.com/sensu/sandbox/blob/master/sensu-go/files/curl_timings-check.json#L10

Can we make the connection more clear?

Reply:
OK -- so you don't have to open `curl_timings-check.json` at all in this step?

Reply:
I'm thinking something like...

Use the `curl_timings-check.json` file provided with the sandbox to create a service check that runs the `metrics-curl.rb` check plugin every 10 seconds on all entities with the `entity:sensu-go-sandbox` subscription and sends events to the InfluxDB pipeline: